src/utils/hooks/useKubevirtClusterServiceVersion.ts

```diff
@@ -25,7 +25,7 @@ export const useKubevirtClusterServiceVersion: UseKubevirtClusterServiceVersion
   });
 
   const subscription = useMemo(
-    () => subscriptions?.find((sub) => sub?.spec?.name?.endsWith(KUBEVIRT_HYPERCONVERGED)),
+    () => subscriptions?.find((sub) => sub?.spec?.name === KUBEVIRT_HYPERCONVERGED),
```

Comment:
i think here u have an issue as on openshift the name is `kubevirt-hyperconverged` but in okd is `community-kubevirt-hyperconverged` so u have to revert this and [this](https://github.com/kubevirt-ui/kubevirt-plugin/pull/3279/changes#diff-8f6c5f74f539748948dc587987dfa75e8e8188565c30a090b0af236931034f12R45)


Tests are on okd so this is why its failing on tests but not on our clusters